openshift/github-mirror.yaml

```diff
@@ -34,6 +34,23 @@ objects:
               value: ${GITHUB_USERS}
             - name: GITHUB_MIRROR_URL
               value: ${GITHUB_MIRROR_URL}
+            - name: GITHUB_MIRROR_CACHE_TYPE
+              value: ${GITHUB_MIRROR_CACHE_TYPE}
+            - name: REDIS_ENDPOINT
+              valueFrom:
+                secretKeyRef:
+                    key: db.endpoint
+                    name: ghmirror-elasticache-stage
+            - name: REDIS_PORT
+              valueFrom:
+                secretKeyRef:
+                    key: db.port
+                    name: ghmirror-elasticache-stage
+            - name: REDIS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                    key: db.auth_token
+                    name: ghmirror-elasticache-stage                    
```

Comment:
The secret name should come from parametrization so we can define different names for the secret on stage when compared to prod. The stage one can be the default then.

Reply:
of course! that makes sense!